scripts/recreate_database.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+path="$( cd -- "$( dirname -- "${BASH_SOURCE[0]:-$0}"; )" &> /dev/null && pwd 2> /dev/null; )";
```

Comment:
Please add a brief explanatory comment to this statement.
People with very limited scripting knowledge (such as I am!) take a long time to assess this.

Reply:
Added a brief note to it. Is it sufficient?